mmcv/cnn/bricks/swish.py

```diff
@@ -0,0 +1,24 @@
+import torch
+import torch.nn as nn
+
+from .registry import ACTIVATION_LAYERS
+
+
+@ACTIVATION_LAYERS.register_module()
+class Swish(nn.Module):
+    """ wish Module.
```

Comment:
typo: Swish

Reply:
Thanks. It's fixed in the latest version.